Client/Client.Qt/Pages/RegistrationPage.cpp

```diff
@@ -51,8 +51,8 @@ RegistrationPage::RegistrationPage(QWidget* parent) : Page(parent)
         if (email.empty() || login.empty() || password.empty() || repeatPassword.empty())
         {
             Base::Logger::FileLogger::getInstance().log("some field is empty", Base::Logger::LogLevel::WARNING);
-            _popupSystemMessage->setPopupText("some field is empty");
-            _popupSystemMessage->show();
+            _popupMessage->setPopupText("some field is empty");
```

Comment:
Why do we have there lowercase letter at the start of sentence? I guess it's our new popup message and user will see this if field is empty. So, I don't quite sure that lowercase letter there is cool

Reply:
Because _popupMessage is statement in class field and for users doesn't matter. We use lowercase only for system statements.

Reply:
@memorisecodead Ivan mean text "some field is empty"
I'm also agreed, text should be started with uppercase